cmake/Modules/FindLibRealSense.cmake

```diff
@@ -0,0 +1,67 @@
+###############################################################################
+# Find Intel librealsense
+#
+#     find_package(LIBREALSENSE)
+#
+# Variables defined by this module:
+#
+#  LIBREALSENSE_FOUND                 True if librealsense was found
+#  LIBREALSENSE_INCLUDE_DIRS          The location(s) of librealsense headers
+#  LIBREALSENSE_LIBRARIES             Libraries needed to use librealsense
+
+find_package(PkgConfig QUIET)
+
+# Include directories
+find_path(LIBREALSENSE_INCLUDE_DIR librealsense2/rs.h
+          PATHS "$ENV{realsense2_DIR}"
+                "$ENV{PROGRAMW6432}/librealsense2" # for self built library 
+                "$ENV{PROGRAMFILES}/librealsense2" # for self built library
+                "$ENV{PROGRAMFILES}/Intel RealSense SDK 2.0" # for pre built library
+                "$ENV{LIBREALSENSE_ROOT}"
+                # "Please specify search paths for Linux and MacOS"
```

Comment:
Should it add <code>"/usr/local/"</code> and <code>"/usr/"</code> for Linux and MacOS?

Reply:
Seems reasonable, and looking at libusb, these seem to be some of the standard hints on linux: https://github.com/PointCloudLibrary/pcl/blob/46cb8fe5589e88e36d79f9b8b8e5f4ff4fceb5de/cmake/Modules/Findlibusb-1.0.cmake

I'll do a quick review of the librealsense docs and see if they install to any of these paths by default on linux/mac